dask/compatibility.py

```diff
@@ -56,10 +60,26 @@ def _getargspec(func):
     range = xrange
     reduce = reduce
     operator_div = operator.div
+    FileNotFoundError = IOError
```

Comment:
I'd prefer to just use the standard error rather than invent our own.  Python2 users will almost expect a plain `IOError` when handling exceptions.  They might be surprised by a special error type.